.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: minimal
+
+services:
+  - docker
+
+jobs:
+  include:
+    - stage: main
+      name: "RPMlint"
+      before_install:
+        - docker pull quay.io/hairmare/fedora_rpmdev
+      script: docker run --rm -ti -v `pwd`:'/git' quay.io/hairmare/fedora_rpmdev rpmlint liquidsoap.spec
```

Comment:
It would be possible, yes... OTOH I've been using and recommending this for a while and don't plan on stopping to maintain it. Given that it's already all over the place it seems like it would lead to some duplication if I move it to the radiorabe orga.

Reply:
Understood. This definitely shouldn't be a show stopper for this PR, but I would like to see such integral parts of our build infrastructure in a place were more than one person can give access to.

Reply:
I sent you a contributor invite to https://github.com/hairmare/docker-rpmdev to ensure I'm not the only one that can push to the repo. I'm not sure if it's as easy to share access to the quay.io project but it should be all set up for the time being.

In the long term I plan on automating some of the travis setup on all the centos-rpm-* repos. It'll be easier to replace this container with one in the @radiorabe orga once that is done.